### DISCUSSION TITLE
viewpoint follow inside a node

### DISCUSSION THREAD

**Discussion by haishuishan** [pelicanmapping/osgearth#1899] [Support Q&A] [answered]
Now I use osg::ref_ptr<osgEarth::Util::EarthManipulator>em, setting a viewpoint inside a plane node,like sitting on the plane. I use setTetherNode() to viewpoint follow the plane.But when the plane is running,the viewpoint is outside the node.not inside the plane.I want to know how change to make the viewpoint follow till inside the node when the node is running?How can I do that?

**Comment by gwaldron** [comment, marked as answer]
Look at the members of `Viewpoint` and see what you need to change. There is a `range` member that is the distance from the camera to the tether node, and a `positionOffset` member for local XYZ offsets. 

Failing all that you may need to use a different camera manipulator to realize a real cockpit view.
